agent-bootstrap/src/main/java/io/opentelemetry/auto/bootstrap/instrumentation/decorator/HttpServerTracerBase.java

```diff
@@ -0,0 +1,243 @@
+/*
```

Comment:
Renamed /instrumentation/decorator/HttpServerTracer.java to /instrumentation/decorator/HttpServerTracerBase.java and removed all references to the Tracer field